### PR TITLE
Create manual_test_rule_template

### LINKED ISSUE #1952: Manual Tests Rule Creation Group
We have a lot of rules missing for the manual tests. 

I am proposing we create a sub group for those who are interested in helping to write the tests. These will be less technical than the automated tests and I hope to make a start on a few. (Well 1…)

Please let me know what you think, and who is interested in joining. We could meet once a month to discuss pain points or ideas separate from the main group discussions?

### PR DESCRIPTION
**Pull request by HelenBurge**
Adding a first go at the manual test rule template (same as the others but with extra guidance at the start)

Starting on the work for the new manual test rules 

Closes issue(s):

Need for Call for Review:
This will not require a Call for Review - it is a template

---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by WilcoFiers — APPROVED** [review]

**Review by Jym77 — APPROVED** [review]
The file should be moved together with the other templates.

File: manual_test_rule_template [hunk 1]
```diff
@@ -0,0 +1,107 @@
+This is the template for the manual test rules. It matches the automation rules but there are parts that can be completed with help (like adding examples of code).
```
Comment (carlosapaduarte) [inline manual_test_rule_template]:
```suggestion
This is the template for the manual test rules. It matches the automation rules but there are parts that can be completed with help from other community members (like adding examples of code).
```

**Review by carlosapaduarte — APPROVED** [review]
With a small suggestion. Feel free to leave it out.

**Comment by Jym77** [comment]
There is a problem with the tests 🙈 
Due to the text at the start, before the first `---`, the frontmatter is not seen by the parsing (`.use(remarkFrontmatter)` in `getMarkdownData`). In turn, this means that
* `prettier` tries to re-pretties it and break the correct indentation it should have (mildly annoying).
* spell check tries to spell-check the frontmatter and complains about `forConformance` or similar keys (very annoying since it breaks tests and we cannot merge).

Easy solutions:
* Move instructions inside (or after) the frontmatter. This may be a bit more annoying for rule writers, but would solve both problems.
* Add spelling exceptions. A bit dirty but working to solve the second problem. The first stays.
* Ignore this file in `describe-page.js` and in the prettier hook. Could also work; not sure how to ignore it in the prettier hook, though.

@HelenBurge Do you think that moving the instructions inside, or after the frontmatter (after the first or second `---`) would be OK? Essentially, there cannot be anything before the first `---` in the file.

**Comment by HelenBurge** [comment]
> There is a problem with the tests 🙈 Due to the text at the start, before the first `---`, the frontmatter is not seen by the parsing (`.use(remarkFrontmatter)` in `getMarkdownData`). In turn, this means that
> 
> * `prettier` tries to re-pretties it and break the correct indentation it should have (mildly annoying).
> * spell check tries to spell-check the frontmatter and complains about `forConformance` or similar keys (very annoying since it breaks tests and we cannot merge).
> 
> Easy solutions:
> 
> * Move instructions inside (or after) the frontmatter. This may be a bit more annoying for rule writers, but would solve both problems.
> * Add spelling exceptions. A bit dirty but working to solve the second problem. The first stays.
> * Ignore this file in `describe-page.js` and in the prettier hook. Could also work; not sure how to ignore it in the prettier hook, though.
> 
> @HelenBurge Do you think that moving the instructions inside, or after the frontmatter (after the first or second `---`) would be OK? Essentially, there cannot be anything before the first `---` in the file.

I have amended the instructions to be after the ---

File: pages/design/manual-template-empty.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  -  # (e.g. HTTP Messages, DOM Tree, CSS Styling, Accessibility Tree, Language, etc.,)
+acknowledgments:
+  authors:
+    -  # (full names as found in `contributors` property in `package.json` - if not yet listed, please have authors added to the list)
+  previous_authors:
+    -  # (full names as found in `contributors` property in `package.json` - if not yet listed, please have authors added to the list)
+---
+This is the template for the manual test rules. It matches the automation rules but there are parts that can be completed with help from other community members (like adding examples of code).
+
+For the starting section, and examples, complete what you feel comfortable with and then ask for help via emails to the chairs of the ACT Task Force and Community Group. Delete these instructions after the --- and before "id:" (delete rows 38-41).
```
Comment (Jym77) [inline pages/design/manual-template-empty.md]:
```suggestion
For the starting section, and examples, complete what you feel comfortable with and then ask for help via emails to the chairs of the ACT Task Force and Community Group. Delete these instructions after the --- and before "## Applicability" (delete rows 38-42).
```

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
Changes proposed by Jean-Yves need to be applied

**Review by carlosapaduarte — APPROVED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Pull request merged**